integration/substrate/package.json

```
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "tsc; ts-mocha -t 20000 *.spec.ts",
-    "build": "parallel solang compile -v --target substrate ::: *.sol test/*.sol",
+    "build": "parallel solang compile -v --target substrate --report-errors=true --math-overflow=true::: *.sol test/*.sol",
```

Please always make sure it works locally first

```suggestion
    "build": "parallel solang compile -v --target substrate --report-errors=true --math-overflow=true ::: *.sol test/*.sol",
```